<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flightCard</class>
 <widget class="QWidget" name="flightCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#flightCard{
border: 1px solid #000000; 
border-radius: 10px;
background-color: #ffffff;
padding: 10px;
}
</string>
  </property>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="departimeLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#departimeLabel{
font-size: 20px; 
font-weight: bold
}</string>
   </property>
   <property name="text">
    <string>出发时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label{
	color: rgb(186, 186, 186);
}</string>
   </property>
   <property name="text">
    <string>—————————————</string>
   </property>
  </widget>
  <widget class="QLabel" name="arrivalLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#arrivalLabel{
font-size: 20px; 
font-weight: bold
}</string>
   </property>
   <property name="text">
    <string>到达时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#price{
	color: rgb(255, 0, 0);
	font-size: 20px;
	font-weight: bold
}</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buyPushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#buyPushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#buyPushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#buyPushButton:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
   </property>
   <property name="text">
    <string>购票</string>
   </property>
  </widget>
  <widget class="QLabel" name="airlineLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>281</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>航空公司|航班号</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
